Source/WebCore/rendering/RenderElement.cpp

```diff
@@ -1712,6 +1712,13 @@ bool RenderElement::allowsAnimation() const
     return page().imageAnimationEnabled();
 }
 
+bool RenderElement::useSystemDarkAppearance() const
+{
+    if (RefPtr page = document().page())
```

Comment:
Do we need to consult style here, like `computeTextPaintStyle()` does?

Reply:
I do not think we need to do that. All we need is to transfer the system dark appearance to the SVGImage. I do not think the style of the RenderImage which references the SVGImage should affect the styles inside the SVGImage.